frontend_vue/src/utils/utils.ts

```diff
@@ -12,3 +12,16 @@ export function convertUnixTimeStampToDate(unixTimestamp: number) {
   const formatter = new Intl.DateTimeFormat('en-US', options);
   return formatter.format(date);
 }
+
+export const MAX_IMG_SIZE = 102400;
```

Comment:
where is this coming from? I think that we actually configure this on the backend